src/utils/buildTimeRangeOptions.js

```diff
@@ -0,0 +1,22 @@
+const computeTimeRangeLabel = (value) => {
+  const multipleDays = value > 1;
+  const dayCount = multipleDays ? `${value} ` : '';
+  const pluralForm = multipleDays ? 's' : '';
+  return `Last ${dayCount}day${pluralForm}`;
+};
+
+// eslint-disable-next-line
+const buildTimeRangeOptions = (values, newValue) => {
```

Comment:
Thanks a lot for the advice! I thought more about the case in which someone might change the url to a value that doesn't exist in the TIME_RANGE_OPTIONS constant. So I went ahead and created this util. I've made changes based on your feedback but I would like to preserve this functionality. Hope it's ok. I'm working also on a maximum value limit, to avoid crashes.

Thanks again! I'll update this asap.